white_rabbit/urls.py

```diff
@@ -40,6 +40,11 @@
         views.EstimatedDaysCountView.as_view(),
         name="estimated-days-reporting",
     ),
+    path(
+        "suivi-monnaitaire",
```

Comment:
```suggestion
        "suivi-monétaire",
```
(oui on a le droit aux accents, j'aime bien perso)